### DISCUSSION TITLE
SB5.1 - OCI - Deploy/UCI/Inbound failed

### DISCUSSION THREAD

**Discussion by heungheung** [Sunbird-Ed/Community#374] [Installation] [answered]
Inbound deployment failed with Jenkins console message
```
TASK [sunbird-deploy : copy and template env file from swarm role] *************
failed: [localhost] (item=../../../../ansible/roles/stack-sunbird/templates/inbound.env) => {
    "ansible_loop_var": "item",
    "changed": false,
    "item": "../../../../ansible/roles/stack-sunbird/templates/inbound.env"
}

MSG:

AnsibleUndefinedVariable: 'uci_odk_user' is undefined
```

environment of uci odk file
https://github.com/project-sunbird/sunbird-devops/blob/release-5.1.0/ansible/roles/stack-sunbird/templates/inbound.env#L35-L36

looks like there should be somewhere to define the uci_odk_user and uci_odk_password, shall these be in the core inventory, or there will be another layer? Based on
https://uci.sunbird.org/use/developer/development-environment/environment-variable-setup
and 
https://docs.getodk.org/aggregate-digital-ocean/
seems the user and password should be defined and get populated to DB.

Did we missed something?

**Comment by beepdot** [comment]
@saiakhil46 Please check this and raise PR to templates if required.

**Reply by saiakhil46** [reply]
@heungheung we are adding these details in our template, we will share you the details shortly.

**Reply by saiakhil46** [reply]
@heungheung This was answered in https://github.com/Sunbird-Ed/Community/discussions/373 discussion please refer to it

**Comment by Krishnaj20** [comment, marked as answer]
@heungheung - marking this as answered as this is a duplicate of the query asked in https://github.com/orgs/Sunbird-Ed/discussions/373
